plugin/evm/vm_test.go

```diff
@@ -967,6 +967,37 @@ func testConflictingImportTxs(t *testing.T, genesis string) {
 	}
 }
 
+// newExportTx returns a new ExportTx
+func (vm *VM) newExportTx(
```

Comment:
Wait we can define methods in test files on non-test structs? TIL 🤷 !
nit  If it's only for tests, maybe pass in a `t *testing.T` and use `require.NoError(t, err)` within the function to not have to return an error (removes a bunch of if err != nil I suppose).

Reply:
yea it's a bit weird, just removed it.